src/Components/Components/src/Rendering/Renderer.cs

```diff
@@ -28,6 +28,7 @@ public abstract partial class Renderer : IDisposable
         private bool _isBatchInProgress;
         private ulong _lastEventHandlerId;
         private List<Task> _pendingTasks;
+        private bool _disposed;
```

Comment:
Didn't we agree that we were making this concept available to subclasses?

Reply:
Ryan mentioned it as an idea, and we can do so if we want to.

Currently I don't think it's necessary. Subclasses can track their disposed state if they want to, and we can add a new protected API for this later if we want to.